src/pip/_internal/req/constructors.py

```diff
@@ -71,11 +71,21 @@ def parse_editable(editable_req):
         - a requirement name
         - an URL
         - extras
-        - editable options
```

Comment:
This line seems stale. The function was already returning a 3-tuple before this change.

Reply:
Indeed, req_file.py does not support per-requirement options for editables.